tests/purs/layout/BacktickOperator.out

```diff
@@ -0,0 +1,16 @@
+module Test where{
+
+example1 = do{
+  foo bar}
+  <|> baz;
+
+example2 = do{
+  foo bar}
+  `wat` baz;
+
+example3 =
+  case _ of{
+    Foo a -> 1;
+    Bar b -> 2}
+    `append` 3}
```

Comment:
This isn't right, is it? First, the braces don't match; second, I'd expect
```suggestion
  {case _ of
    Foo a -> 1;
    Bar b -> 2}
    `append` 3
```

(which of course wouldn't type check, because `append` would need type `(a -> Int) -> Int -> b`, but that's not really relevant to a layout test).

Compare with how
```purs
example3 =
  case _ of
    Foo a -> 1
    Bar b -> 2
    + 3
```
currently parses.

Reply:
That last `}` is the layout end of the module, I think.

Reply:
I agree we should have a comparable test here for case with operators, since there is one for `do`.

Reply:
:facepalm: Of course it is. Okay, guess this all makes sense after all!

Reply:
I've added a case for operators.